MAINTAINERS.md

```diff
@@ -0,0 +1,24 @@
+# ODPi Maintainers
+## What is an ODPi Maintainer?
+
+ODPi Maintainers are the key leaders in the development of the ODPi specifications. They are key members of the ODPi community who have shown both leadership and collaboration skills in thier work within the community.
+
+### Roles and Responisibilities
+
+* Review spec and test harness contributions that for the spec(s) you are a maintainer for.
```

Comment:
This sentence is incoherent.  I think it means, "Review spec and test harness contributions for the spec(s) you are a maintainer for.", but I'm not 100% certain that's the intent.